interfaces/Portal/src/app/program/registration-activity-detail-accordion/registration-activity-detail-accordion.component.html

```diff
@@ -74,7 +74,9 @@
             <!-- sent -->
             <div
               class="util--half-padding-vertical util--half-margin-end"
-              style="border-bottom: 1px solid var(--fiveten-neutral-200)"
+              style="
+                border-bottom: 1px solid var(--ion-color-fiveten-neutral-300);
```

Comment:
We could replace this repeated line with a (local to this component) class like "`border-bottom--light`"? (or a better name)